File: leetcode-arry/26. 删除有序数组中的重复项/java/RemoveDuplicatesFromSortedArray.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length == 1) {
            return 1;
        }

        int size = 0;
        int len = nums.length;
        for(int i = 0; i < len; i++) {
            while(i + 1 < nums.length && nums[i] == nums[i + 1] && len > 1) {
                int var;
                for(int j = i + 1;j < nums.length - 1; j++) {
                    var = nums[j + 1];
                    nums[j + 1] = nums[j];
                    nums[j] = var;
                }
                len--;
            }
            size++;
        }
        return size;
    }
}

File: leetcode-arry/26. 删除有序数组中的重复项/python/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        left = right = 1
        for i in range(1,len(nums)):
            if nums[right] != nums[right-1]:
                nums[left] = nums[right]
                left +=1
            right +=1
        return left